contracts/programs/ocr2/src/lib.rs

```diff
@@ -278,25 +278,23 @@ pub mod ocr2 {
     }
 
     #[access_control(proposal_owner(&ctx.accounts.proposal, &ctx.accounts.authority))]
-    pub fn propose_payees(
-        ctx: Context<ProposeConfig>,
-        token_mint: Pubkey,
-        payees: Vec<Pubkey>,
-    ) -> Result<()> {
+    pub fn propose_payees(ctx: Context<ProposeConfig>, token_mint: Pubkey) -> Result<()> {
```

Comment:
Yeah it's not because we'd have to specify up to 19 old accounts to pay out the oracles + 19 new token accounts belonging to the new token: over the limit

Reply:
https://github.com/smartcontractkit/chainlink-solana/issues/395